src/funcx_common/messagepack/message_types/container.py

```diff
@@ -0,0 +1,21 @@
+import typing as t
+import uuid
+
+from .base import Message, meta
+
+
+@meta(message_type="containerimage")
+class ContainerImage(Message):
+    image_type: str
+    location: str
+    created_at: int
+    modified_at: int
+    build_status: t.Optional[str]
+    build_stderr: t.Optional[str]
+
+
+@meta(message_type="container")
+class Container(Message):
+    container_id: uuid.UUID
+    name: str
+    images: t.List[ContainerImage]
```

Comment:
I'm not super familiar with the container service, so I won't know what fields like `location` or `image_type` mean without tracing them out in the services/SDK/endpoint. Should we add comments here?

Reply:
I was on the fence about it, erring toward not.  I'll jump to the other side of the fence.